Document a known issue for IBM Power and Z (#927)
quay/quay-docs: modules/rn_3_11_0.adoc

If you are using the the automatic mode selection, which chooses between light or dark modes depending on the user's browser preference, your operating system appearance is overridden by the browser website appearance setting. If you find that the device-based theme is not working as expect, check your browser appearance setting. This is a known issue and will be fixed in a future version of {productname}. (link:https://issues.redhat.com/browse/PROJQUAY-6903[*PROJQUAY-6903*])

[id="IBM-P-and-Z-known-issue"]

> ==== UI Build Log issue for IBM Power & IBM Z

Reviewer: ```suggestion
=== UI Build Log issue for IBM Power & IBM Z
```

Author: Ack. Thanks.

> UI Build Log issue for IBM Power & IBM Z. UI V2 Build feature is not supported with ODF. It works fine with AWS S3 or any other external object storage. This is a known issue and will be fixed in a future version of {productname}. (link:https://issues.redhat.com/browse/PROJQUAY-6940[*PROJQUAY-6940*]).

Reviewer: ```suggestion
The UI V2 Build feature is not supported with {odf}. It works with AWS S3 or any other external object storage. This is a known issue and will be fixed in a future version of {productname}. (link:https://issues.redhat.com/browse/PROJQUAY-6940[*PROJQUAY-6940*]).
```

Author: Ack. Thanks.
